.github/workflows/Checkov.yml

```diff
@@ -0,0 +1,22 @@
+name: Checkov
+on:
+  workflow_dispatch:
+  push:
```

Comment:
We'll want to run this on pull_request as well so that "bad" code doesn't make it into main